### DISCUSSION TITLE
Framing engraving light lines around project

### DISCUSSION THREAD

**Discussion by Katrinka725** [arkypita/LaserGRBL#2145] [Q&A]
Could someone please help me fix a problem. I’m new to engraving and when I frame it is causing lines where I have framed. Thank you for any help you can give. 
![IMG_3982](https://github.com/arkypita/LaserGRBL/assets/148909774/3c79ddd1-a1ed-4182-9722-d50993a225ef)

**Comment by Tribo-2dot0** [comment]
Repetindo aqui a resposta do outro tópico...

Clique com o direito sobre o botão Frame > Editar Botão, diminua o valor de S na segunda linha.
Podes deixar a segunda linha assim: M3 S5 F1000
Se o laser não acender aumente para S6... S7... etc. Se quiser que faça a moldura mais rápido, aumente o valor de F.
